buoy_gazebo/src/PolytropicPneumaticSpring/PolytropicPneumaticSpring.cpp

```diff
@@ -211,7 +260,9 @@ void PolytropicPneumaticSpring::Configure(
   igndbg << "is upper? " << std::boolalpha << this->dataPtr->is_upper << std::noboolalpha <<
     std::endl;
 
-  this->dataPtr->valve_absement = SdfParamDouble(_sdf, "valve_absement", 5e-6);
+  this->dataPtr->valve_absement = SdfParamDouble(_sdf, "valve_absement", 49e-7);
```

Comment:
Can you pass `this->dataPtr->valve_absement` here to avoid repeating the default value? The same for other lines

```suggestion
  this->dataPtr->valve_absement = SdfParamDouble(_sdf, "valve_absement", this->dataPtr->valve_absement);
```

Reply:
I'm fixing all of this in #55